# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/ugly-no-nptl.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- glibc-20050919/nscd/nscd.h.vanilla	2005-11-05 14:47:31.000000000 +0100
+++ glibc-20050919/nscd/nscd.h	2005-11-05 14:47:17.000000000 +0100
@@ -25,6 +25,7 @@
 #include <stdbool.h>
 #include <time.h>
 #include <sys/uio.h>
+#include <sys/socket.h>
 
 /* The declarations for the request and response types are in the file
    "nscd-client.h", which should contain everything needed by client
--- glibc-20050919/malloc/memusage.c.vanilla	2005-11-05 13:54:54.000000000 +0100
+++ glibc-20050919/malloc/memusage.c	2005-11-05 13:56:00.000000000 +0100
@@ -83,7 +83,12 @@
 static memusage_cntr_t decreasing_mremap;
 static memusage_size_t current_heap;
 static memusage_size_t peak_use[3];
+
+#ifdef HAVE___THREAD
 static __thread uintptr_t start_sp;
+#else
+static uintptr_t start_sp;
+#endif
 
 /* A few macros to make the source more readable.  */
 #define peak_heap	peak_use[0]
